lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -701,6 +694,21 @@ sub jsonrpc_validate {
                 }
             };
         }
+
+        my %locale = $self->{config}->Language_Locale_hash();
+
+        # NOTE: Default to first language in config
+        my $language = $jsonrpc_request->{params}->{language} // $self->{config}->{_LANGUAGE_locale}[0];
```

Comment:
It could be OK that the first `locale` i the default, but then it should be documented.

Another comment is that what comes over RPCAPI is normally a two-character language code, e.g. "en", whereas `$self->{config}->{_LANGUAGE_locale}[0]` will return a language-country string, e.g. "en_US".

Reply:
> It could be OK that the first locale i the default, but then it should be documented.

Yes I have planned to update the doc on that matter.

> Another comment is that what comes over RPCAPI is normally a two-character language code, e.g. "en", whereas $self->{config}->{_LANGUAGE_locale}[0] will return a language-country string, e.g. "en_US".

The local hash returned by `Language_Locale_hash` support both two chars language code and language contry string.

```perl
{
          'fr_FR' => 'fr_FR.UTF-8',
          'sv' => 'sv_SE.UTF-8',
          'fi' => 'fi_FI.UTF-8',
          'sv_SE' => 'sv_SE.UTF-8',
          'fi_FI' => 'fi_FI.UTF-8',
          'en_US' => 'en_US.UTF-8',
          'fr' => 'fr_FR.UTF-8',
          'da' => 'da_DK.UTF-8',
          'nb' => 'nb_NO.UTF-8',
          'da_DK' => 'da_DK.UTF-8',
          'nb_NO' => 'nb_NO.UTF-8',
          'en' => 'en_US.UTF-8'
};
```